### PR TITLE
CSI-1818 add 1.4.0 CSV yamls

### PR DESCRIPTION
**Pull request by oriyarde**

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,766 @@
...
+
+  keywords:
+    - IBM
+    - BlockStorage
+    - CSI
+  version: 1.4.0
+  replaces: ibm-block-csi-operator.v1.3.0
+  maturity: stable
+  maintainers:
+    - name: Guang Jiong Lou
```
Comment (ArbelNathan) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:412]: is he?

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:412]: IMO, ideally, we should find a shared email alias to put here

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,766 @@
...
+    $ kubectl apply -f demo-storageclass-gold-pvc.yaml
+    storageclass.storage.k8s.io/gold created
+    ```
+
+  keywords:
+    - IBM
+    - BlockStorage
+    - CSI
+  version: 1.4.0
+  replaces: ibm-block-csi-operator.v1.3.0
```
Comment (roysahar-ibm) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:409]: Is this suppose to be 1.3.0?

File: deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,766 @@
...
+    $ kubectl apply -f demo-storageclass-gold-pvc.yaml
+    storageclass.storage.k8s.io/gold created
+    ```
+
+  keywords:
+    - IBM
+    - BlockStorage
+    - CSI
+  version: 1.4.0
+  replaces: ibm-block-csi-operator.v1.3.0
```
Comment (roysahar-ibm) [inline deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:409]: Suppose to be 1.3.0?

**Review by roysahar-ibm — COMMENTED** [review]
reviewing

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml (hunk 2 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:409]: yes, 1.4.0 replaces 1.30

File: deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml (hunk 3 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:409]: yes, 1.4.0 replaces 1.30

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml (hunk 1 above)
Comment (ArbelNathan) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:412]: do you want to do it in this scope or to open a ticket?

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:412]: I opened CSI-2144

**Review by roysahar-ibm — APPROVED** [review]

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**